tooling/ast_fuzzer/minimizer/scripts/docker-entry.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+# We assume that there is a `main.nr` and a `Prover.toml` file mounted in the working directory,
+# which `cvise` is going to copy to a temporary directory, each time with some new minimization.
+
+# Create a check script which creates a new project with the contents we want to test,
+# run a `nargo` command, and checks the presence of the error message in the output.
+# If it's not present, it means the latest reduction step was not interesting.
+cat > check.sh <<EOF
+nargo new test_project
+cp main.nr test_project/src
+cp /noir/Prover.toml test_project
+cd test_project
+nargo $CMD 2>&1 | grep "$MSG"
+EOF
```

Comment:
I went back and forth whether this should be a script checked into source control, taking env vars, without having the message verbatim in it. But then I thought `cvise` docs say we should provide a script that takes no parameters, and this is more in line with that.